cookbook/workflow/homestead.rst

```diff
@@ -0,0 +1,66 @@
+.. index:: Vagrant, Homestead
+
+Using Symfony with Homestead/Vagrant
+====================================
+
+In order to develop a Symfony application, you might want to use a virtual
+development environment instead of the built-in server or WAMP/LAMP. Homestead_
+is an easy-to-use Vagrant_ box to get a virtual environment up and running
+quickly.
```

Comment:
Many of the linked issues are related to someone saying "Hey, I tested this on a clean Vagrant box", which is exactly why Vagrant was created in the first place. I can't see many issues directly related to Vagrant bugs.

And following some guides from Benjamin's blog can speed up applications very nicely. In fact, on my Windows PC, applications run on Homestead are much faster and consume less CPU than those running on the built-in PHP webserver or Wamp.